### PR TITLE
feat(): added graphql request

### PR DESCRIPTION
**Pull request by Sigwoman**
- added graphql request
- added investigation static info query

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,5 +23,6 @@
     "concurrently": "^5.3.0"
   },
   "dependencies": {
+    "body-parser": "^1.19.0"
```
Comment (RoeiRom) [inline package.json:26]: Move to server directory

File: server/package.json [hunk 2]
```diff
@@ -15,11 +15,14 @@
     "@graphile/pro": "^0.10.0",
     "@types/pg": "^7.14.4",
     "cors": "^2.8.5",
+    "@types/request-promise": "^4.1.46",
```
Comment (RoeiRom) [inline server/package.json:18]: Move to Dev dependencies

File: server/src/ClientToDBAPI/mainRoute.ts [hunk 3]
```diff
@@ -1,17 +1,24 @@
...
 import exposureRoute from './ExposureRoute/mainRoute';
 import landingPageRoute from './LandingPageRoute/mainRoute';
+import investigationInfo from './InvestigationInfo/mainRoute';
 import intersectionsRoute from './IntersectionsRoute/mainRoute';
 import personalDetailsRoute from './PersonalDetailsRoute/mainRoute';
 import clinicalDetailsRoute from './ClinicalDetailsRoute/mainRoute';
 
 const clientToDBRouter = Router();
 
+clientToDBRouter.post('/*', bodyParser.json(), (req, res, next) => {
```
Comment (RoeiRom) [inline server/src/ClientToDBAPI/mainRoute.ts:13]: You can use the body parser to the server main file, the server

File: server/src/ClientToDBAPI/mainRoute.ts [hunk 4]
```diff
@@ -1,17 +1,24 @@
...
+    next();
+});
+
 clientToDBRouter.use('/exposure', exposureRoute);
 clientToDBRouter.use('/landingPage', landingPageRoute);
 clientToDBRouter.use('/intersections', intersectionsRoute);
+clientToDBRouter.use('/investigationInfo', investigationInfo);
+clientToDBRouter.use('/presonalDetails', personalDetailsRoute);
```
Comment (RoeiRom) [inline server/src/ClientToDBAPI/mainRoute.ts:21]: personalDetails*

File: server/src/MOHAPI/mainRoute.ts [hunk 5]
```diff
@@ -6,4 +7,8 @@ mohRouter.get('/', (request: Request, response: Response) => {
     response.send('Hello from MOH Api');
 })
 
+mohRouter.post('/*', bodyParser.json(), (req, res, next) => {
```
Comment (RoeiRom) [inline server/src/MOHAPI/mainRoute.ts:10]: The same here

**Review by RoeiRom — CHANGES_REQUESTED** [review]

**Review by EmilyLip — APPROVED** [review]

File: server/src/ClientToDBAPI/InvestigationInfo/mainRoute.ts [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import { Router, Request, Response } from 'express';
+
+import { graphqlRequest } from '../../GraphqlHTTPRequest';
+import { GET_INVESTIGATION_INFO } from '../../DBService/InvestigationInfo/Query';
+
+const investigationInfo = Router();
+
+investigationInfo.post('/staticInfo', (request: Request, response: Response) => {
+    console.log(request.body)
```
Comment (RoeiRom) [inline server/src/ClientToDBAPI/InvestigationInfo/mainRoute.ts:9]: למחוק

File: server/package.json [hunk 7]
```diff
@@ -14,16 +14,20 @@
     "@graphile/pg-pubsub": "^4.8.0",
     "@graphile/pro": "^0.10.0",
     "@types/pg": "^7.14.4",
```
Comment (RoeiRom) [inline server/package.json:16]: נכון שלא שלך אבל להעביר לdev depandancy

**Review by RoeiRom — CHANGES_REQUESTED** [review]

**Review by RoeiRom — APPROVED** [review]

**Review by EmilyLip — APPROVED** [review]

**Pull request merged**